src/librustc_error_codes/error_codes/E0080.md

```diff
@@ -14,7 +14,10 @@ constant expression that had to be evaluated. Attempting to divide by 0
 or causing an integer overflow are two ways to induce this error.
 
 Ensure that the expressions given can be evaluated as the desired integer type.
-See the FFI section of the Reference for more information about using a custom
-integer type:
 
-https://doc.rust-lang.org/reference.html#ffi-attributes
+See the [Custom Discriminants][custom-discriminants] section the Reference for
+more information about setting custom integer types on field-less Enums using
```

Comment:
```suggestion
more information about setting custom integer types on fieldless enums using
```

Reply:
I see the original article has the other spelling, but I think this is the wrong choice. I've opened https://github.com/rust-lang/reference/pull/768 in the meantime.